rest/data/interface.go

```diff
@@ -10,7 +10,7 @@ type Connector interface {
 	FindPerformanceResultById(string) (*model.APIPerformanceResult, error)
 	RemovePerformanceResultById(string) (int, error)
 	FindPerformanceResultsByTaskId(string, util.TimeRange, ...string) ([]model.APIPerformanceResult, error)
-	FindPerformanceResultsByTaskName(string, util.TimeRange, int, ...string) ([]model.APIPerformanceResult, error)
+	FindPerformanceResultsByTaskName(string, string, util.TimeRange, int, ...string) ([]model.APIPerformanceResult, error)
```

Comment:
Should we change the method name to match?

Reply:
I think the method name is still correct since the only required info is the task name